package.json

```diff
@@ -14,23 +14,24 @@
   },
   "homepage": "https://github.com/scality/vaultclient#readme",
   "dependencies": {
-    "arsenal": "scality/Arsenal#67365083",
-    "commander": "2.9.0",
-    "xml2js": "0.4.17",
-    "werelogs": "scality/werelogs#0ff7ec82"
+    "arsenal": "scality/Arsenal#4fd77d73",
+    "commander": "2.20.0",
+    "xml2js": "0.4.19",
+    "werelogs": "scality/werelogs#4e0d97c"
```

Comment:
The correct tags for Werelog are:
development/7.4 - 4e0d97c
development/7.5 - 4e0d97c
development/8.0 - 351a2a3  -> needs to be fixed in the integration branch
development/8.1 - 351a2a3   -> needs to be fixed in the integration branch

Reply:
Updated development/7.4 to 4e0d97c . Will follow up with the integration branches before merging.